credential/src/container.rs

```diff
@@ -27,10 +22,22 @@ impl ProvideAwsCredentials for ContainerProvider {
 
         let address: String = format!("http://{}{}", AWS_CREDENTIALS_PROVIDER_IP, aws_container_credentials_relative_uri);
 
-        let mut response = match reqwest::get(&address)
+        let mut response =
+            match retry::retry_exponentially(5, 0_f64, || { reqwest::get(&address) },
```

Comment:
This number should be greater than zero to do exponential backoff.  A value of 7 gives this:

```
2017/02/15 22:36:55
2017/02/15 22:36:56
2017/02/15 22:36:59
2017/02/15 22:37:07 
2017/02/15 22:37:29 
```

I say we try that for now.